cmd/manager/main.go

```diff
@@ -106,6 +109,14 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := mgr.AddReadyzCheck("ping", healthz.Ping); err != nil {
```

Comment:
It looks like these health and readiness checks will always return "ready" or "healthy". They don't actually check anything, other than verifying that the process can respond to an HTTP request. Is that the intent? It would be a good idea to document the criteria either way, both in the commit message, and also in the README or similar documentation.